config/functions

```diff
@@ -818,7 +818,7 @@ fix_module_depends() {
   local OLD_DEPENDS=""
   cp ${MODULE} ${MODULE}_orig
   $OBJDUMP -s -j .modinfo ${MODULE}_orig | awk 'BEGIN{v=0;} /Contents/ {v=1; next;} {if (v==1) print $0;}' >new.modinfo1
-  cat new.modinfo1 | cut -c7-41 | awk '{printf($0);}' | sed 's/ //g;s/../\\\x&/g;' >new.modinfo2
+  cat new.modinfo1 | cut -c7-41 | awk '{echo($0);}' | sed 's/ //g;s/../\\\x&/g;' >new.modinfo2
```

Comment:
This one shouldn't be changed and does need to be `printf()` - there's no `echo()` function in `awk`.